### PR TITLE
REF: LNNodeBar to TS

### PR DESCRIPTION
**Pull request by Danswar**

### CODE REVIEW

**Review by marcosrdz — APPROVED** [review]

File: components/LNNodeBar.tsx [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+  disabled?: boolean;
+  itemPriceUnit?: BitcoinUnit;
+}
+
+export const LNNodeBar: React.FC<LNNodeBarProps> = ({
+  canReceive = 0,
+  canSend = 0,
+  nodeAlias = '',
+  disabled = false,
+  itemPriceUnit = BitcoinUnit.SATS
```
Comment (limpbrains) [inline components/LNNodeBar.tsx:20]: please run `npm run lint:fix` before commit

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: components/LNNodeBar.tsx (hunk 1 above)
Comment (Danswar) [reply components/LNNodeBar.tsx:20]: hey @limpbrains, I did it, but it is complaining for a problem in a file I didn't touch in this PR 😖

Screenshot 2024-02-20 at 20 52 00

Comment (Danswar) [reply components/LNNodeBar.tsx]: dismiss the previous comment

**Comment by Danswar** [comment]
@limpbrains Thanks for the comment. I think everything should go well this time.

**Review by limpbrains — APPROVED** [review]

**Review by marcosrdz — APPROVED** [review]

**Pull request merged**